pages/linux/bwa.md

```diff
@@ -0,0 +1,24 @@
+# bwa
+
+> BWA is a short, low-divergent DNA sequences mapper against a large reference genome, such as the human genome.
```

Comment:
```suggestion
> Burrows-Wheeler Alignment tool.
> Short, low-divergent DNA sequences mapper against a large reference genome, such as the human genome.
```
How about this? (Referred to their man page at https://bio-bwa.sourceforge.net/bwa.shtml)

Reply:
Their man page is not completely up to date, but I like this desc.